File: trunk/FireflowSuite/engine/src/org/fireflow/engine/IRuntimeContextAware.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package org.fireflow.engine;

/**
 *
 * @author chennieyun
 */
public interface IRuntimeContextAware {
    public void setRuntimeContext(RuntimeContext ctx);
    
    public RuntimeContext getRuntimeContext();
}

File: dot-net_version/FireWorkflow.Net/FireWorkflow.Net/Engine/IRuntimeContextAware.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FireWorkflow.Net.Engine
{
    /// <summary>
    /// 类似IWorkflowSessionAware
    /// </summary>
    public interface IRuntimeContextAware
    {
        void setRuntimeContext(RuntimeContext ctx);

        RuntimeContext getRuntimeContext();
    }
}
